setup.cfg

```diff
@@ -11,9 +11,19 @@ known_first_party =
 default_section = THIRDPARTY
 
 [flake8]
+# Ignoring unused imports since mypy would warn of that.
+ignore = F401
```

Comment:
Sorry I've been quite busy these last months so I'm a little late in my review ^^
Couldn't we tag the unused imports specifically and keep this check in flake8 ?

Reply:
flake8 doesn't actually understand the mypy declarations and shouts about them. mypy (obviously) does and doesn't shout about valid usage.

IIRC, I remember reading somewhere in the mypy documentation that it does detect if a certain import is unused, which is what made me feel comfortable about silencing these warnings. Now that both linters are run one after another, I don't think it's an issue if mypy catches something instead of flake8.

Reply:
Oh, and I had looked at https://github.com/ambv/flake8-mypy -- The first line of the description of that project makes me think this approach is better suited.

Further, the "Two levels of type checking" section of that project's README motivated me to actually take this route.